src/app/screens/SignUp/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import i18next from 'i18next';
 import { useForm } from 'react-hook-form';
 
-import { User } from 'typings/user';
+import { User } from '../../../utils/types';
```

Comment:
`import { User } from '~utils/types';`

remember the absolute routes available